source/setup.py

```diff
@@ -258,6 +258,14 @@ def _genManifestTemplate(shouldHaveUIAccess: bool) -> tuple[int, int, bytes]:
 			) + (
 				"__pycache__",
 		))
-		+ getRecursiveDataFiles('documentation', '../user_docs', excludes=('*.t2t', '*.t2tconf', '*/developerGuide.*'))
+	+ getRecursiveDataFiles(
+		"documentation", "../user_docs",
+		excludes=(
+			"*.t2t", "*.t2tconf", "*.md",
+			"*.py", "__pycache__",
+			"*/user_docs/styles.css",
+			"*/developerGuide.*"
+		)
+	)

```

Comment:
Please put each item on a new line.
It appears styles.css is not being specified correctly - there is no user_docs folder in user_docs. Can you confirm you tested this? Are you sure `styles.css` should not be included?

```suggestion
	+ getRecursiveDataFiles(
		"documentation",
		"../user_docs",
		excludes=(
			"*.t2t",
			"*.t2tconf",
			"*.md",
			"*.py",
			"__pycache__",
			"styles.css",
			"*/developerGuide.*"
		)
	)

```

Reply:
Regarding `user_docs/styles.css`, it seems to match like this:

```python
>>> import fnmatch
>>> print(fnmatch.fnmatch("../user_docs/styles.css", "user_docs/styles.css"))
False
>>> print(fnmatch.fnmatch("../user_docs/styles.css", "*/user_docs/styles.css"))
True
``` 

By the way, the `getRecursiveDataFiles` function is a bit too unintuitive.

```python
def get(dest, source, excludes=()):
	rulesList = []
	for f in glob("%s/*" % source):
		if (
			not any(fnmatch.fnmatch(f, exclude) for exclude in excludes)
			and os.path.isfile(f)
		):
			rulesList.append((
				dest,
				[f]))
	for dirName in os.listdir(source):
		if os.path.isdir(os.path.join(source, dirName)) and not dirName.startswith('.'):
			rulesList.extend(
				getRecursiveDataFiles(
					os.path.join(dest, dirName),
					os.path.join(source, dirName),
					excludes=excludes
				)
			)
	return rulesList
``` 

But this is not exactly equivalent to the previous list derivation, because even if the files are all excluded, there will still be a tuple with an empty list in rulesList.